### ISSUE TITLE
route unique --columns=nexthopIps produces an error

### ISSUE THREAD

**Issue by jopietsch** [netenglabs/suzieq#289]
**Describe the bug**
```(suzieq-QZqWZG2D-py3.8) jpietsch@jpietsch-server:~/suzieq$ suzieq/cli/suzieq-cli route unique --columns=nexthopIps
                                                                                                error
0  ERROR: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```

**Comment by jopietsch** [comment]
and oifs
```
(suzieq-QZqWZG2D-py3.8) jpietsch@jpietsch-server:~/suzieq$ suzieq/cli/suzieq-cli route unique --columns=oifs
                                                                                                error
0  ERROR: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```

**Comment by ddutt** [comment]
Fixed with commit 8a9397a57713cf08a30b0445d0d50aa68d015316
